includes/definitions/discovery/tplink.yaml

```diff
@@ -0,0 +1,4 @@
+modules:
+    os:
+        hardware: RMON-MIB::rmon.19.3.0
```

Comment:
shouldn't this be: RMON2-MIB::probeHardwareRev.0 and RMON2-MIB::probeSoftwareRev.0

Reply:
oh, old code used the partial oids, lol